---
layout: page
permalink: /repositories/
title: Data
nav: true
nav_order: 3
---

<br>

The genomes were sequenced by Oxford nanopore and Illumina. Flye assembler (option: --meta) was usedto assemble the raw ONT reads which were then polished by four iterations of Racon, followed by Medaka. The consensus sequences were further corrected with Illumina reads using NextPolish and haplotigs were removed using Purge Dups. Contigs with non-nematode origins were excluded. 
{: .text-justify}

For annotation, repetitive elements were identified using RepeatModeler, TransposonPSI, and USEARCH based on the protocol by Berriman *et al*. Repetitive DNA sequences were identified and masked using Repeatmasker. The proteomes of 11 representative nematodes were obtained from WormBase. Single worm transcriptome reads were mapped to the corresponding genome assemblies using STAR and assembled using Trinity (guided approach), Stringtie, and Cufflinks. Transcripts generated by Trinity were mapped to the corresponding genome assemblies using Minimap2 (options: -ax splice), and splice junctions were quantified using Portcullis. The gene predictor Augustus and gmhmm were trained using BRAKER2 and SNAP with proteomes and RNAseq mappings as evidence hints to generate an initial set of annotations. The assembled transcripts selected by MIKADO, proteome homology, and BRAKER2 annotations were combined as evidence hints for input into the MAKER2 annotation pipeline to produce a final annotation for each species.
{: .text-justify}

Here, we provide all the assembly, annotation, CDSs, and proteome files as follows. The assembly, CDSs, and proteome files are in FASTA format.
{: .text-justify}

<br>

# *de novo* genome assembly

|---
|Species|Class|Assembly|Annotation|CDSs|Proteome|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Mesodorylaimus* sp.|Dorylaimida|[GENOME][1]|[GFF][2]|[CDS][3]|[PEP][4]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Mesodorylaimus* sp.|Enoplida|[GENOME][5]|[GFF][6]|[CDS][7]|[PEP][8]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Trissonchulus* sp.|Enoplida|[GENOME][9]|[GFF][10]|[CDS][11]|[PEP][12]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Trileptium ribeirensis*|Enoplida|[GENOME][13]|[GFF][14]|[CDS][15]|[PEP][16]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Theristus* sp.|Chromadorea|[GENOME][17]|[GFF][18]|[CDS][19]|[PEP][20]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Sabatieria punctata*|Chromadorea|[GENOME][21]|[GFF][22]|[CDS][23]|[PEP][24]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Ptycholaimellus* sp.|Chromadorea|[GENOME][25]|[GFF][26]|[CDS][27]|[PEP][28]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Linhomoeus* sp.|Chromadorea|[GENOME][29]|[GFF][30]|[CDS][31]|[PEP][32]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Paralinhomoeus* sp.|Chromadorea|[GENOME][33]|[GFF][34]|[CDS][35]|[PEP][36]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Microlaimidae* sp.|Chromadorea|[GENOME][37]|[GFF][38]|[CDS][39]|[PEP][40]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Epsilonema* sp.|Chromadorea|[GENOME][41]|[GFF][42]|[CDS][43]|[PEP][44]|
|:-:|:-:|:-:|:-:|:-:|:-:|
|*Rhynchonema* sp.|Chromadorea |[GENOME][45]|[GFF][46]|[CDS][47]|[PEP][48]|
|---

<br><br>

# Raw Reads Accession
The Oxford nanopore sequcing and Illumina seqqencing reads of genome and Illumina seqqencing reads of transcriptome has been deposed in the NCBI under bioproject [PRJNA953805](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA953805). 


[2]: {{ site.url }}/Data/Annotation_gff/Enoplolaimus_lenunculus.gff
