<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Data | Single nematode genome  </title>
    <meta name="author" content="Single nematode genome  ">
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/Single-nematode-genome/assets/css/main.css">
    <link rel="canonical" href="https://yichienlee1010.github.io/Single-nematode-genome/repositories/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/Single-nematode-genome/assets/js/theme.js"></script>
    <script src="/Single-nematode-genome/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/Single-nematode-genome/"><span class="font-weight-bold">Single nematode genome </span></a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/Single-nematode-genome/">About</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/Single-nematode-genome/publications/">Publications</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/Single-nematode-genome/projects/">Project</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/Single-nematode-genome/repositories/">Data<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/Single-nematode-genome/cv/">cv</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/Single-nematode-genome/teaching/">Authors</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Data</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <p><br></p>

<p class="text-justify">The genomes were sequenced by Oxford nanopore and Illumina. Flye assembler (option: –meta) was usedto assemble the raw ONT reads which were then polished by four iterations of Racon, followed by Medaka. The consensus sequences were further corrected with Illumina reads using NextPolish and haplotigs were removed using Purge Dups. Contigs with non-nematode origins were excluded.</p>

<p class="text-justify">For annotation, repetitive elements were identified using RepeatModeler, TransposonPSI, and USEARCH based on the protocol by Berriman <em>et al</em>. Repetitive DNA sequences were identified and masked using Repeatmasker. The proteomes of 11 representative nematodes were obtained from WormBase. Single worm transcriptome reads were mapped to the corresponding genome assemblies using STAR and assembled using Trinity (guided approach), Stringtie, and Cufflinks. Transcripts generated by Trinity were mapped to the corresponding genome assemblies using Minimap2 (options: -ax splice), and splice junctions were quantified using Portcullis. The gene predictor Augustus and gmhmm were trained using BRAKER2 and SNAP with proteomes and RNAseq mappings as evidence hints to generate an initial set of annotations. The assembled transcripts selected by MIKADO, proteome homology, and BRAKER2 annotations were combined as evidence hints for input into the MAKER2 annotation pipeline to produce a final annotation for each species.</p>

<p class="text-justify">Here, we provide all the assembly, annotation, CDSs, and proteome files as follows. The assembly, CDSs, and proteome files are in FASTA format.</p>

<p><br></p>

<h1 id="de-novo-genome-assembly">
<em>de novo</em> genome assembly</h1>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Species</th>
      <th style="text-align: center">Class</th>
      <th style="text-align: center">Assembly</th>
      <th style="text-align: center">Annotation</th>
      <th style="text-align: center">CDSs</th>
      <th style="text-align: center">Proteome</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Mesodorylaimus</em> sp.</td>
      <td style="text-align: center">Dorylaimida</td>
      <td style="text-align: center">[GENOME][1]</td>
      <td style="text-align: center"><a href="https://YiChienLee1010.github.io/Data/Annotation_Gff/Enoplolaimus_lenunculus.gff">GFF</a></td>
      <td style="text-align: center">[CDS][3]</td>
      <td style="text-align: center">[PEP][4]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Mesodorylaimus</em> sp.</td>
      <td style="text-align: center">Enoplida</td>
      <td style="text-align: center">[GENOME][5]</td>
      <td style="text-align: center">[GFF][6]</td>
      <td style="text-align: center">[CDS][7]</td>
      <td style="text-align: center">[PEP][8]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Trissonchulus</em> sp.</td>
      <td style="text-align: center">Enoplida</td>
      <td style="text-align: center">[GENOME][9]</td>
      <td style="text-align: center">[GFF][10]</td>
      <td style="text-align: center">[CDS][11]</td>
      <td style="text-align: center">[PEP][12]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center"><em>Trileptium ribeirensis</em></td>
      <td style="text-align: center">Enoplida</td>
      <td style="text-align: center">[GENOME][13]</td>
      <td style="text-align: center">[GFF][14]</td>
      <td style="text-align: center">[CDS][15]</td>
      <td style="text-align: center">[PEP][16]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Theristus</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][17]</td>
      <td style="text-align: center">[GFF][18]</td>
      <td style="text-align: center">[CDS][19]</td>
      <td style="text-align: center">[PEP][20]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center"><em>Sabatieria punctata</em></td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][21]</td>
      <td style="text-align: center">[GFF][22]</td>
      <td style="text-align: center">[CDS][23]</td>
      <td style="text-align: center">[PEP][24]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Ptycholaimellus</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][25]</td>
      <td style="text-align: center">[GFF][26]</td>
      <td style="text-align: center">[CDS][27]</td>
      <td style="text-align: center">[PEP][28]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Linhomoeus</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][29]</td>
      <td style="text-align: center">[GFF][30]</td>
      <td style="text-align: center">[CDS][31]</td>
      <td style="text-align: center">[PEP][32]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Paralinhomoeus</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][33]</td>
      <td style="text-align: center">[GFF][34]</td>
      <td style="text-align: center">[CDS][35]</td>
      <td style="text-align: center">[PEP][36]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Microlaimidae</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][37]</td>
      <td style="text-align: center">[GFF][38]</td>
      <td style="text-align: center">[CDS][39]</td>
      <td style="text-align: center">[PEP][40]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Epsilonema</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][41]</td>
      <td style="text-align: center">[GFF][42]</td>
      <td style="text-align: center">[CDS][43]</td>
      <td style="text-align: center">[PEP][44]</td>
    </tr>
  </tbody>
  <tbody>
    <tr>
      <td style="text-align: center">
<em>Rhynchonema</em> sp.</td>
      <td style="text-align: center">Chromadorea</td>
      <td style="text-align: center">[GENOME][45]</td>
      <td style="text-align: center">[GFF][46]</td>
      <td style="text-align: center">[CDS][47]</td>
      <td style="text-align: center">[PEP][48]</td>
    </tr>
  </tbody>
</table>

<p><br><br></p>

<h1 id="raw-reads-accession">Raw Reads Accession</h1>
<p>The Oxford nanopore sequcing and Illumina seqqencing reads of genome and Illumina seqqencing reads of transcriptome has been deposed in the NCBI under bioproject <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA953805" rel="external nofollow noopener" target="_blank">PRJNA953805</a>.</p>


          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2023 Single nematode genome  . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/Single-nematode-genome/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/Single-nematode-genome/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/Single-nematode-genome/assets/js/common.js"></script>
  <script defer src="/Single-nematode-genome/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
