After whole genome amplification and nanopore sequencing, we found palindrome sequencing in reads affected the assembly quality. Therefore, we established this script - remove_palindrome.sh, it is combined with Minimap2 and Perl scripts to treat the palindromic sequences. Through Minimap2 alignments, we identified artificial palindromic sequences, described as reads that map to the reverse complement version of themselves. We recognize the position of the palindromic sequences in reads using the Perl script- pafIdentifyPalimdrom.pl and with Perl script - fastq_partition.and.chop.palindrome.pl, these palindromic sequences were extracted from raw reads and corrected by dividing the read from the midpoint of the alignment. We recommend performing this process in two iterations as a sequence may encompass multiple copies of the original fragment.



